src/components/Footer.jsx

```diff
@@ -9,52 +9,38 @@ const Footer = () => {
         <div className="footerin">
           <div className="w40">
             <h3>Join Our Community</h3>
-            <a href="https://www.google.com" title="Join Now">
-              Join Now
+            <a
+              target="_blank"
+              href="https://forum.mycovidconnect.com/"
+              title="Join Now"
+            >
+              <button>Join Now</button>
             </a>
           </div>
           <div className="w30">
-            <ul>
-              <li>
-                <Link to="/">Contact Us</Link>
-              </li>
-              <li>
-                <Link to="/">FAQ</Link>
-              </li>
-            </ul>
-          </div>
-          <div className="w30">
-            <ul>
-              <li>
-                <Link to="/" title="About XCOV19">
-                  About XCOV19
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Work with Us">
-                  Work with Us
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Privacy Policy">
-                  Privacy Policy
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Terms &amp; Conditions">
-                  Terms &amp; Conditions
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Press Enquiries">
-                  Press Enquiries
-                </Link>
-              </li>
-            </ul>
+            <div className="links">
+              <a
+                target="_blank"
+                href="https://forum.mycovidconnect.com/d/2-about-xcov19-rapid-action-patient-dispatch-solution"
+              >
```

Comment:
add `target="_blank" rel="noopener"` in all links we want the [user to open externally](https://mathiasbynens.github.io/rel-noopener/) wherever there is `target="_blank"`

Also Add a todo over here somewhere like this:
 `TODO(@vasynizami) create a common js script for external links asking for conset`
Idea is like:
```javascript
 () => {
  // suspends initial behavior and asks user consent Yes/No whether they want to navigate to external llink or not? if not suspends default behaviour else continues. 
}
```